common/app/layout/Breakpoint.scala

```diff
@@ -0,0 +1,19 @@
+package layout
+
+import scalaz.syntax.std.option._
+
+sealed trait Breakpoint {
+  val minWidth: Option[Int]
+}
+
+case object Mobile extends Breakpoint {
+  val minWidth = None
+}
+
+case object Tablet extends Breakpoint {
+  val minWidth = 740.some
```

Comment:
We have a list of breakpoints in [javascript](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/utils/detect.js#L26) [css](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/_vars.scss#L48) and now here.

Maybe put a comment in each of those places to help people keep them in sync.